usecases/schema/add_property.go

```diff
@@ -45,8 +46,10 @@ func (m *Manager) addClassProperty(ctx context.Context,
 	}
 	prop.Name = lowerCaseFirstLetter(prop.Name)
 
-	m.setNewPropDefaults(class, prop)
-
+	err = m.setNewPropDefaults(class, prop)
+	if err != nil {
+		return err
+	}
```

Comment:
small style suggestion, feel free to ignore

```
	if err = m.setNewPropDefaults(class, prop); err != nil {
		return err
	}
```

Reply:
Fixed.